substrate/client/authority-discovery/src/worker.rs

```diff
@@ -174,6 +180,8 @@ pub struct Worker<Client, Block: BlockT, DhtEventStream> {
 
 	metrics: Option<Metrics>,
 
+	warn_public_addresses: bool,
```

Comment:
A comment on what this does would be nice. I first thought this should be exposed, until I had read the code :D